frontend/src/App.js

```diff
@@ -19,34 +19,46 @@ function App() {
 const[themeList, setThemeList]=useState([
     {id:"test1",
         themeName:"Bla",
-        basicList:[{id:"b1",name:"Defnition",content:"XXXXXXXYYYYY"}],
-    exampleList:[],
-        homeworkList:[]}]);
-  //  state mit dem Themen
+        basicList:[{id:"b1",name:"Defnition",content:"XXXXXXXYYYYY"},
+                   {id:"b2",name:"Defnition2",content:"WWWWWW"}],
+        exampleList:[{id:"ex1",subtopic:"Negation",example:"123123123"},
+                     {id:"ex2",subtopic:"Konjunktion",example:"456456456"}],
+        homeworkList:[{id:"hw1",subtopic:"Aussagen",topic:"Beurteilen Sie...",
+                        result:true,solution:"Lösung: yyyy"},
+                      {id:"hw2",subtopic:"Negation_und_Äquivalenz",
+                          topic:"Beurteilen Sie den Wahrheitsgehalt...",
+                              // "Gegeben seien die falsche Aussage <TeX>{\"A\"}</TeX> und die wahre Aussage <TeX>{\"B\"}</TeX>.\n" +
+                              // "Beurteilen Sie den Wahrheitsgehalt der Aussage <TeX>{\"A\\\\vee(\\\\neg B)\"}</TeX>.",
+                        result:false,solution:"Lösung: xxxx"}]},
+    {id:"test2",
+        themeName:"lalala",
+        basicList:[{id:"b2_1",name:"Defnition3",content:"xxxxx"},
+            {id:"b2_2",name:"Defnition4",content:"yyyyy"}],
+        exampleList:[{id:"ex3",subtopic:"kartesische_Produkt",example:"aaaaaaa"},
+            {id:"ex4",subtopic:"Konjunktion",example:"bbbbbb"}],
+        homeworkList:[{id:"hw2_1",subtopic:"Mengen",topic:"Gegeben seien...",
+                        result:true,solution:"Lösung: AAAAA"},
+                       {id:"hw1",subtopic:"Aussagen",topic:"Beurteilen Sie...",
+                         result:false,solution:"Lösung: BBBBB"}]}
+]);
+
```

Comment:
Die Aufgaben hier zu erstellen wird in Zukunft etwas unübersichtlich werden, vermute ich...